tests/system/Bigtable/BigtableInstanceAdminTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
```

Comment:
2018 :) - Looks to be an issue in all files

Reply:
haha thanks

Reply:
Fixed!